machine/v1beta1/types_azureprovider.go

```diff
@@ -83,6 +86,11 @@ type AzureMachineProviderSpec struct {
 	// SecurityProfile specifies the Security profile settings for a virtual machine.
 	// +optional
 	SecurityProfile *SecurityProfile `json:"securityProfile,omitempty"`
+	// UltraSSDCapability enables or disables Azure UltraSSD capability for a virtual machine.
+	// When omitted, the platform may enable the capability based on the configuration of data disks.
+	// +kubebuilder:validation:Enum:="Enabled";"Disabled"
+	// +optional
+	UltraSSDCapability AzureUltraSSDCapabilityState `json:"ultraSSDCapability,omitempty"`
```

Comment:
Why would a customer want to enable ultradisk capability on a per machineset basis?

Reply:
See response to previous question, to allow PVs to use UltraSSD